README.md

```diff
@@ -37,19 +37,21 @@ configuration:
 
 ```hcl-terraform
 module "ecs_cluster" {
-  source = "infrablocks/ecs-cluster/aws"
-  version = "3.4.0"
+  source = "github.com/infrablocks/terraform-aws-ecs-cluster"
```

Comment:
This changes the source from the Terraform registry to the HEAD of the Github repository. I don't think this should be included and instead, the version should be updated to the latest release version which is "4.2.0".

Reply:
Sounds good, I just updated it to use the registry pointer. @tobyclemson